community/kernel/src/main/java/org/neo4j/storageengine/api/StorageStatement.java

```diff
@@ -112,9 +113,10 @@
      -     */
     Cursor<RelationshipItem> relationshipsGetAllCursor();
 
-    Cursor<PropertyItem> acquirePropertyCursor( long propertyId, Lock shortLivedReadLock );
+    Cursor<PropertyItem> acquirePropertyCursor( long propertyId, Lock shortLivedReadLock, AssertOpen assertOpen );
```

Comment:
In 3.2 you can no longer get a property cursor directly from a node cursor. Instead you explicitly grab a property cursor by providing node or relationship object and use `nextPropertyId` to initialise a new cursor in `StorageLayer`.